### ISSUE TITLE
Prohibit Git rebase in the docs

### ISSUE THREAD

**Issue by mohammad-sdsc** [SwissDataScienceCenter/renku#880]
## Description

Rewriting git history can cause problem for Renku. Make sure that the documentation is clear about prohibition of rewriting the Git history.

**Comment by rokroskar** [comment]
We probably can't prohibit it - but we should build the KG so that rebasing doesn't cause problems.

**Comment by ableuler** [comment]
> we should build the KG so that rebasing doesn't cause problems.

I agree. I rewrite history on non-protected branches all the time. Not being able to do this would be a huge limitation.
